src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -366,14 +430,14 @@ export class OnlineGameWrapperComponent extends GameWrapper<MinimalUser> impleme
         this.gameComponent.updateBoard();
     }
     public switchPlayer(): void {
-        display(OnlineGameWrapperComponent.VERBOSE, 'OnlineGameWrapperComponent.switchPlayer');
+        display(OnlineGameWrapperComponent.VERBOSE, 'OnlineGameWrapperComponent.switchPlayer at turn ' + this.currentPart.data.turn);
         const part: PartDocument = this.currentPart;
         const currentPlayer: Player = Player.fromTurn(part.data.turn);
         this.currentPlayer = this.players[this.gameComponent.rules.node.gameState.turn % 2].get();
         const currentOpponent: Player = currentPlayer.getOpponent();
-        if (this.didUserPlay(currentOpponent)) {
-            this.pauseCountDownsFor(currentOpponent);
-        }
+        // if (this.didUserPlay(currentOpponent)) {
+        this.pauseCountDownsFor(currentOpponent);
+        // }
```

Comment:
commented code should die

Reply:
https://www.youtube.com/watch?v=EiqFcc_l_Kk